.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+sudo: false
+go:
```

Comment:
Yeah, the only downside of tip is it takes a lot longer to fetch and can be flaky at times